macros/EmbedInteractiveExample.ejs

```diff
@@ -10,7 +10,7 @@
 let jsEditorClass = "";
 let surveyClass = "interactive-editor-survey";
 let surveyLink = "https://www.surveygizmo.com/s3/3780959/MDN-Interactive-Editor";
-let url = "https://interactive-examples.mdn.mozilla.net/" + $0;
+let url = kuma.url.resolve(env.interactive_examples.base_url, $0);
```

Comment:
I'm getting an error:

```
Cannot read property 'base_url' of undefined
    at eval (eval at compile (/node_modules/ejs/lib/ejs.js:491:12), <anonymous>:21:52)
    at returnedFn (/node_modules/ejs/lib/ejs.js:520:17)
    at /app/lib/kumascript/templates.js:71:26
```

Adding some debug statements to see what is going on...

Reply:
@jwhitlock Do you get this when you run `make test-macros`? Hmm, my first guess is that you may need to rebuild your kumascript image `cd .. && make build-kumascript`?

Reply:
No, the macros tests explictly set ``env.interactive_examples``, so the error doesn't occur. It only occurs when using the value set by ``server.js``